index.html

```diff
@@ -289,11 +289,57 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-    </main>
 
-    <script
-      src="https://kit.fontawesome.com/e966a3bd22.js"
-      crossorigin="anonymous"
-    ></script>
+
+    <footer id="Contact">
+      <section class="head">
+        <h2>
+          I'm always <br />
+          interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+      </section>
+
+      <form class="form" action="https://formspree.io/f/xdobnpad" method="post">
+        <input type="text" id="name" name="user-name" maxlength="30" placeholder="Full Name" required />
+        <input type="text" id="email" name="user-email" maxlength="500" placeholder="Email Address" required />
```

Comment:
- [ ] Your form looks pretty good :sparkles: :sparkles: However, the email field should be of type `email`. This will automatically validate the input value to ensure it is a properly formatted e-mail address :+1: 
As you can see in the screenshot below :point_down:  when `type="email"l`, the email field only accepts emails
![Screenshot from 2022-05-20 11-11-06](https://user-images.githubusercontent.com/87965534/169497266-de8194ac-d2ce-445b-91ee-000949ee401b.png)
- [ ] [OPTIONAL] The attribute `maxlength="500"` isn’t necessary on the Email input, usually email addresses are short. Plus, there are technical limitations to how long email addresses can be, you can check this link if [you like to know more about it.](https://blog.moonmail.io/what-is-the-maximum-length-of-a-valid-email-address-f712c6c4bc93)
:smile: